### DISCUSSION TITLE
Top of multi step form

### DISCUSSION THREAD

**Discussion by dmganly** [Budibase/budibase#6202] [Help]
Hi guys,

While I have managed to make real progress in all aspects of multi step form creation, with particular thanks to https://budibase.com/blog/app-building/public-forms-how-to-build-them-in-minutes/, I am wondering when the Next or Previous form Step is opened, is there a way of bringing focus to the top most field, where the user would next be inputting info.  This would make the UX swing.  Some of my form Steps have five or six large text areas, so moving to the next Step at the moment just brings me to the middle of the form which knocks me flat.  Any ideas most welcome.  Thanks.

**Comment by melohagan** [comment]
There is no way to do this at the moment, but the feature has been requested here: https://github.com/Budibase/budibase/issues/3671

It would be nice!
